src/groovy/org/transmartproject/db/dataquery/highdim/snp_lz/SnpLzModule.groovy

```diff
@@ -171,7 +175,11 @@ class SnpLzModule extends AbstractHighDimensionDataTypeModule {
                 property 'snp.trialName',              'trialName'
 
```

Comment:
It looks like the DeSnpInfo table contains that info related to SNPs. Am I mistaken?